test/framework/cloudstack.go

```diff
@@ -38,6 +38,8 @@ const (
 	cloudstackSshAuthorizedKeyVar      = "T_CLOUDSTACK_SSH_AUTHORIZED_KEY"
 	cloudstackTemplateRedhat123Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_23"
 	cloudstackTemplateRedhat124Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_24"
+	cloudstackTemplateRedhat125Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_25"
+	cloudstackTemplateRedhat126Var     = "T_CLOUDSTACK_TEMPLATE_REDHAT_1_26"
```

Comment:
we need to add these template variables to buildspecs for e2e tests and conformance tests (cmd/integration_test/build/buildspecs/cloudstack-test-eks-a-cli.yml and cmd/integration_test/build/buildspecs/conformance-test-eks-a-cli.yml)